Fix issue template support.md (#5603)
prometheus-operator/prometheus-operator: .github/ISSUE_TEMPLATE/support.md

---
name: Support

> about: If you have questions about the Prometheus Operator. Helm? Go to https://github.com/prometheus-community/helm-charts. kube-prometheus ? Go to https://github.com/prometheus-operator/kube-prometheus

Reviewer: ```suggestion
about: For questions about prometheus-operator. For helm, go to https://github.com/prometheus-community/helm-charts. For kube-prometheus, go to https://github.com/prometheus-operator/kube-prometheus.
```

Reviewer: (nit) I'm good either way

Reviewer 2: I like the suggestion just because it reads better but I'm also fine wither way

Reviewer: Updated the suggestion :)
